components/infobox/wikis/stormgate/infobox_building_custom.lua

```diff
@@ -167,15 +173,33 @@ end
 ---@param args table
 ---@return string[]
 function CustomBuilding:getWikiCategories(args)
-	if not self.faction then
-		return {}
-	end
-
-	return {Faction.toName(self.faction) .. ' Buildings'}
+	local factionName = Faction.toName(self.faction)
+	return Array.append({},
+		factionName and (factionName .. ' Buildings') or nil,
+		args.informationType == CREEP and 'Camps' or nil
+	)
 end
 
 ---@param args table
 function CustomBuilding:setLpdbData(args)
+	if args.informationType == CREEP then
```

Comment:
i think we can deduplicate here
like push the additional extradata into the normal extradata
same for type

Reply:
do you want to "merge" this into the regular storage?

Reply:
yes

Reply:
done